yarn.lock

```diff
@@ -2288,6 +2295,19 @@ elliptic@^6.0.0, elliptic@^6.2.3, elliptic@^6.4.0:
     minimalistic-assert "^1.0.0"
     minimalistic-crypto-utils "^1.0.0"
 
+elliptic@^6.4.1:
```

Comment:
We should delete all the `elliptic` entries in yarn.lock, then re-run yarn, to consolidate versions.